silverback/runner.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, app: SilverBackApp, *args, **kwargs):
         logger.info(f"Using {self.__class__.__name__}: max_exceptions={self.max_exceptions}")
 
     async def _block_task(self, block_handler: AsyncTaskiqDecoratedTask):
-        async for block in async_wrap_iter(chain.blocks.poll_blocks()):
+        async for block in async_wrap_iter(chain.blocks.poll_blocks(new_block_timeout=self.app.new_block_timeout)):
```

Comment:
thinking it may be nice to default to way larger numbers for local networks out-of-the-box